packages/rest-api-client/src/platform/node.ts

```diff
@@ -56,6 +73,14 @@ export const buildPlatformDependentConfig = ({
   return {};
 };
 
+export const buildTimeoutConfig = (params: { socketTimeout?: number }) => {
```

Comment:
IMO: Now, this function doesn't need to be exported.

```suggestion
const buildTimeoutConfig = (params: { socketTimeout?: number }) => {
```